.github/workflows/release.yml

```diff
@@ -1,23 +1,53 @@
 name: Release
 on:
-  push:
-    tags:
-      - v*
+  release:
+   types: [published]
 jobs:
   release:
     runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+
     steps:
-    - uses: actions/checkout@v3
-    - uses: actions/setup-node@v3
-      with:
-        node-version: 16
-    - run: yarn install
-    - run: yarn build
-    - run: 'yarn release'
-      env:
-        CLIENT_ID: ${{ secrets.PUBLISHER_CHROME_CLIENT_ID }}
-        CLIENT_SECRET: ${{ secrets.PUBLISHER_CHROME_CLIENT_SECRET }}
-        REFRESH_TOKEN: ${{ secrets.PUBLISHER_CHROME_REFRESH_TOKEN }}
-        WEB_EXT_API_KEY: ${{ secrets.WEB_EXT_API_KEY }}
-        WEB_EXT_API_SECRET: ${{ secrets.WEB_EXT_API_SECRET }}
-        EXTENSION_ID: 'jlmafbaeoofdegohdhinkhilhclaklkp'
+      - name: Get version from tag
+        id: tag_name
+        run: echo ::set-output name=current_version::${GITHUB_REF#refs/tags/v}
+        shell: bash
+
+      - name: Checkout repository
+        uses: actions/checkout@v3.0.0
+
+      - name: Install node
+        uses: actions/setup-node@v3.1.0
+        with:
+          node-version: 16
+
+      - name: Get changelog entry
```

Comment:
We don't have a change log, so this step isn't needed unless we want to start doing that. If we do it has to be in this format for the action to work (most of the repos listed in my profile readme use this format since I'm also using this action with them) https://keepachangelog.com

Reply:
Thanks, I'll remove it for now.